wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/RuntimeAttributeElement.scala

```diff
@@ -0,0 +1,3 @@
+package wdl.model.draft3.elements
+
+case class RuntimeAttributeElement(key: String, value: String) extends LanguageElement
```

Comment:
ToL: I confused myself the first time I read this (I thought it was referring to the section) - could we add a comment or rename it something like `RuntimeAttributeEntryElement`?

Reply:
I took the names as they are after being parsed and added "Element". I'll rename as suggested.